bootstrap.sh

```diff
@@ -17,4 +17,4 @@ kubectl apply -f .infrastructure/app/deployment.yml
 
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
-# kubectl apply -f .infrastructure/ingress/ingress.yml
+kubectl apply -f .infrastructure/ingress/ingress.yml
```

Comment:
Ensure that the URL for the Ingress Controller deployment is correct and up-to-date. This is crucial for the Ingress to function properly. If there are any issues with the Ingress, verify this URL first.